### DISCUSSION TITLE
Close this section (Github Discussions)?

### DISCUSSION THREAD

**Discussion by fdcastel** [FirebirdSQL/firebird#7570] [Ideas]
Given that this section is [not being](https://github.com/FirebirdSQL/firebird/discussions/7551#discussioncomment-5764111) actively monitored, even [confusing](https://github.com/FirebirdSQL/firebird/discussions/7477#discussioncomment-5011037) some users I want to _frantically_ ask to **close it right now**.

To make one thing perfectly clear: I'm not a fan of mailing lists and consider Github Discussions a very superior alternative to mailing lists of 30 years ago. 

That said, having a place where users can ask a question and got nothing for _one week_ (been there, done that) will not help the project in any capacity.

Also, there has been [very few activity](https://github.com/FirebirdSQL/firebird/discussions?discussions_q=) around here (just 5 entries in almost one year).

Related: https://github.com/FirebirdSQL/firebird/discussions/7228#discussioncomment-3108246

Ping: @mrotteveel

**Comment by mrotteveel** [comment]
The other posts got a reply on the same day. Yours seems to have either fallen through the cracks (I myself had missed the email update from GitHub, or mistaken it for a bug report). So, your experience is N=1, and not a solid reason to change things. 

That said, I personally don't like having two places to discuss things ([firebird-devel Google Group](https://groups.google.com/g/firebird-devel) and this location). I'll raise it with the other admins as well.

**Comment by aafemt** [comment]
For discussions there must be
a) Subject for discussion;
b) People ready to discuss it.

I wouldn't expect any discussions on subjects that are too trivial, plain stupid or not related to Firebird at all. I also think that this section is easy to fix up with issues and is better to be closed.

**Reply by AlexPeshkoff** [reply]
Mark, wait a day or 2 and please close it in a case there are no objections.

**Reply by mrotteveel** [reply]
An objection was raised on firebird-admin by Adriano
